README.md

```diff
@@ -34,6 +34,11 @@ cd apache
 Use the copier template for the latest framework of Salt Extensions. This is the preferred way to create a new extension,
 as there is a lot of shared code between extensions that is easier to keep in sync with copier.
 
+Install copier
+```bash
+pip install copier
```

Comment:
Note that the template requires `copier-templates-extensions` as well.

Reply:
Does that require a separate install or does the next line that is running copier with the `copier-templates-extensions` argument handle that?

Reply:
Nah, it's a separate Python library that needs to be pulled from PyPi. I'll do a quick PR. :)

Reply:
:+1: